binaries/pom.xml

```diff
@@ -34,6 +35,8 @@
 	<modules>
 		<module>org.eclipse.swt.cocoa.macosx.x86_64</module>
 		<module>org.eclipse.swt.cocoa.macosx.aarch64</module>
+		<module>org.eclipse.swt.gtk.freebsd.aarch64</module>
```

Comment:
As these will stay "shell" bundles without actual binaries in them we should not include them so such useless bundles don't get published in the p2 repo. I would be fine if you include them via Maven "freebsd" profile though.

Reply:
Ditto, per my reply above